app/views/research_sessions/data.html.erb

```diff
@@ -1,18 +1,18 @@
 <%= render "research_sessions/progress" %>
 <div class="main-content">
   <%= render 'error_summary' %>
-  <h1 class="subtitle-large">Data</h1>
+  <h1 class="subtitle-large">Storing information</h1>
```

Comment:
If we're going to say "storing" here, and "storing" in the progress bar, we should probably rename this to `storing.html.erb` and also update the wizard [`Steps`](https://github.com/barnardos/consent-form-builder-rails/blob/master/app/models/research_session/steps.rb#L14) to keep our domain language consistent.

Reply:
It looks like if we update [`Steps`](https://github.com/barnardos/consent-form-builder-rails/blob/master/app/models/research_session/steps.rb#L14) as suggested above, that will also change the text in the [progress bar](https://github.com/barnardos/consent-form-builder-rails/blob/master/app/views/research_sessions/_progress.html.erb#L26) – we should probably keep this stuff in line in this PR 😄

Reply:
The progress bar part is covered by different card where all the navigation changes are included

Reply:
That's fine but the step name should probably still remain consistent with the copy in this PR. That would mean the progress bar changing at the same time as a side-effect.